### PR TITLE
Feature: Added a function to scroll into dynamic layer/inner window

### PR DESCRIPTION
**Pull request by snehith07**
created a scroll function that works for mobile & tablet. which does normal scroll & scrolls the inner window

### CODE REVIEW

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 1]
```diff
@@ -482,4 +483,18 @@ public void injectMediaToBrowserstackDevice(String uploadFileURL) {
                     "injectMediaToBrowserstackDevice is not implemented for: " + cloudName);
         }
     }
+
+    public static void scrollInDynamicLayer(Driver driver, String direction) {
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:487]: why is this static?

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 2]
```diff
@@ -482,4 +483,18 @@ public void injectMediaToBrowserstackDevice(String uploadFileURL) {
...
+        Dimension dimension = driver.getInnerDriver().manage().window().getSize();
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:488]: why do you need to pass Driver to a method in the Driver?

File: src/main/java/com/znsio/teswiz/runner/Driver.java [hunk 3]
```diff
@@ -482,4 +483,18 @@ public void injectMediaToBrowserstackDevice(String uploadFileURL) {
...
+        int width = (int) (dimension.width * 0.5);
+        int fromHeight = (int) (dimension.height * 0.7), toHeight = (int) (dimension.height * 0.6);
+        int[] height = {fromHeight, toHeight};
+        if (direction.equalsIgnoreCase("up"))
```
Comment (anandbagmar) [inline src/main/java/com/znsio/teswiz/runner/Driver.java:492]: add { } around if

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 1 above)
Comment (snehith07) [reply src/main/java/com/znsio/teswiz/runner/Driver.java]: removed static keyword

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 2 above)
Comment (snehith07) [reply src/main/java/com/znsio/teswiz/runner/Driver.java]: removed now

File: src/main/java/com/znsio/teswiz/runner/Driver.java (hunk 3 above)
Comment (snehith07) [reply src/main/java/com/znsio/teswiz/runner/Driver.java]: updated

**Pull request merged**